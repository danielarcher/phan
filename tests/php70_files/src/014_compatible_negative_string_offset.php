<?php 
$var = "abcdef"[-2];

$string = 'bar';
echo $string[-1];