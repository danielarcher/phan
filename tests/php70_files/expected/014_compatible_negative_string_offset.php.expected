%s:2 PhanCompatibleNegativeStringOffset Using negative string offsets is not supported before PHP 7.1 (emits an 'Uninitialized string offset' notice)
%s:5 PhanCompatibleNegativeStringOffset Using negative string offsets is not supported before PHP 7.1 (emits an 'Uninitialized string offset' notice)
